### DISCUSSION TITLE
https the connection is not secure

### DISCUSSION THREAD

**Discussion by McMike4** [community/community#137081] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am trying to get access to my localhost. But the https message tells me that the connection is not secure written in red.

I used this url to rsolve the issue : chrome://flags/#allow-insecure-localhost

But it sill doesn't work. Anyone could try to help me?
Thanks

**Comment by phuyalgaurav** [comment]
### Honestly, for localhost, just stick to http://. HTTPS is kind of overkill when you're running things locally—you're not really at risk since it’s all happening on your machine.

But if you really need HTTPS for whatever reason (maybe you're trying to replicate a production setup), you can try a few things:

- Self-Signed Cert: If you're dead set on HTTPS, you can roll with a self-signed cert. It’ll still throw a warning, but at least you can say you tried.

- Chrome Flags: You already found chrome://flags/#allow-insecure-localhost, which is good. Make sure you actually restarted Chrome after turning that on, though. Also, clear your cache—it’s like a browser's memory of all the times you messed up.

- SSL Tunnels: If you're feeling fancy, something like ngrok can give you an HTTPS tunnel to localhost. A bit extra, but hey, it works.

If none of that helps, might be worth taking a step back and asking if you really need HTTPS for local dev. But if you do, good luck!

**Reply by McMike4** [reply]
What I've got in the screenshot is similar to a Self-Signed Cert? 
![non secure https](https://github.com/user-attachments/assets/731f626e-2717-4640-b233-ab71869760c7)

**Reply by phuyalgaurav** [reply]
Yes, You need to configure wampserver to serve your site over https. You should look at the wampserver docs on how to do so.

**Reply by phuyalgaurav** [reply]
But, i think you should take a step back here and just use http instead of https in the url. But, i dont know your specific use case.

**Reply by McMike4** [reply]
Do you know what's the problem in the screenshot and how I could resolve this?

It is said that 'localhost is not recognized as an internal or an external command'. 

Thanks in advance
![localhost not recognized](https://github.com/user-attachments/assets/a9c81806-2c93-46e0-80d9-bd9e9079b16b)

**Reply by metaverseguy** [reply]
The error message you're seeing indicates that the command `localhost` is not recognized as a valid command in the Windows Command Prompt. This is because `localhost` is not a command; it's a hostname that refers to your local machine.

To access a local server (like a WordPress site) using the command line, you typically need to use a web browser or a tool like `curl` or `wget`. Here’s how you can do it:

1. **Open a Web Browser**: Simply type `http://localhost/sitetest/wp-json` in the address bar of your web browser and press Enter.

2. **Using Command Line Tools**:
   - If you want to use the command line, you can use `curl` (if installed) like this:
     ```bash
     curl http://localhost/sitetest/wp-json
     ```
   - If you don't have `curl`, you can install it or use `wget` if it's available.

3. **Check Your Server**: Ensure that your local server (like XAMPP, WAMP, or MAMP) is running and that the site is correctly set up in the server's directory.
